guide/controller-plugin.md

```diff
@@ -301,3 +301,23 @@ A complete sample (communicating with the e-puck robot using bluetooth) can be
 found in this directory:
 
 `WEBOTS_HOME/projects/robots/e-puck/plugins/remote_controls/e-puck_bluetooth`
+
+### HTML robot window
+
+As an alternative to native robot windows described earlier in this section, it is also possible to implement a HTML robot window. HTML robot windows present several advantages over the native robot windows:
+
+1. They are easier to implement: simple HTML layout, simple Javascript programming, no compilation needed.
```

Comment:
When reading this doc, I have the feeling to read a pros list of using it over the old system (I'm speaking about the keywords `easy`, `very simple`, etc.) while a new user just want to read what system he should use and how to use it.
Why not to document this new system first, and the old system secondly, indicating in one sentence that the new system should be preferred?